services/authoring/src/repositories/policy-repository.ts

```diff
@@ -1,30 +1,56 @@
 import Transactor from '../utils/transactor';
 import GitRepository from './git-repository';
 import { Oid } from 'nodegit';
-import { JsonValue } from '../utils/jsonValue';
-import Author from '../utils/author';
+import { Author, JsonValue } from '../utils';
 import jsonpatch = require('fast-json-patch');
 
-export default class PolicyRepository {
+interface Resource {
+  path: string;
+  isKey: boolean;
+}
+
+const getPathToPolicyFile = (resource?: Resource) => {
+  if (!resource) {
+    return 'security/policy.json';
+  }
+
+  return resource.isKey
+    ? `manifests/${resource.path}.json`
+    : `manifests/${resource.path}/policy.json`;
+};
+
+export class PolicyRepository {
```

Comment:
I like more this style of exporting because if we'd like to export something else in the future from this file you won't have to update all the other exports.

Reply:
You can export a default along with more things, but it is a bit clunky to call it